Resources/Textures/Interface/Actions/actions_ai.rsi/meta.json

```diff
@@ -33,6 +33,30 @@
 		},
 		{
 			"name": "comms_console"
+		},
+		{
+			"name": "emergency_off"
+		},
+		{
+			"name": "emergency_on"
+		},
+		{
+			"name": "bolt_door"
+		},
+		{
+			"name": "unbolt_door"
+		},
+		{
+			"name": "electrify_door"
+		},
+		{
+			"name": "unelectrify_door"
+		},
+		{
+			"name": "door_overcharge_on"
```

Comment:
New ones are CC0-1.0 by ScarKy0, don't really know how to add this info to license and copyright! :(

Reply:
inserted new files and mention side-by-side... folder fits too well...